src/main/features/dashboard/views/claimant.njk

```diff
@@ -51,7 +51,7 @@
           </li>
           <li>
             <strong class="bold">{{ t('Claim details:') }}</strong>
-            <span class="block"><a href="{{ ClaimPaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download claim') }}</a></span>
+            <span class="block"><a href="{{ ResponsePaths.claimDetailsPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('View claim') }}</a></span>
```

Comment:
`ResponsePaths` in claimant's dashboard?

Reply:
On response journey, defendant need to view claim details